packages/camera/camera_android_camerax/android/src/test/java/io/flutter/plugins/camerax/InstanceManagerTest.java

```diff
@@ -42,35 +42,132 @@ public void addHostCreatedInstance() {
   }
 
   @Test
-  public void addHostCreatedInstance_createsSameInstanceTwice() {
+  public void remove() {
```

Comment:
Consider either naming this such that what is being tested or adding documentation for the same effect.

Reply:
@bparrishMines also fyi about making this change!